tools/crmadmin.8.inc

```diff
@@ -0,0 +1,5 @@
+[synopsis]
+crmadmin [options] [node]
+
+/the Pacemaker controller/
+.SH OPTIONS
```

Comment:
You'll need to add crmadmin.8.inc to EXTRA_DIST in tools/Makefile.am.

Reply:
Done.